openedx/core/djangoapps/credit/services.py

```diff
@@ -104,6 +109,13 @@ def get_credit_state(self, user_id, course_key_or_id, return_course_name=False):
             result.update({
                 'course_name': course.display_name,
             })
+
+        if return_is_self_paced_course:
```

Comment:
Can we combine the `return_course_name` and `return_is_self_paced_course` flags to a single `return_course_info` flag? Not sure how extensive a change it would be to update references to `return_course_name`, but if it isn't too bad, I think it is worth doing.